app/routes/application.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+const { RSVP: { hash } } = Ember;
+
+export default Ember.Route.extend({
+  model() {
+    return hash({
```

Comment:
You don't need to fetch everything anymore. You can just fetch exactly what you need for each route.